torch/csrc/jit/codegen/cuda/interface.cpp

```diff
@@ -125,7 +125,7 @@ bool complyWith(
         // we are not looking at dim-j, but dim-sorted_index, which
         // is the j-th fastest dim;
         // TODO: merge this with above and put a long comment there
-        if (t_strides[sorted_index] < t_strides[inner_dim]) {
+        if ((t_strides[sorted_index] != 0) and (t_strides[sorted_index] < t_strides[inner_dim])) {
```

Comment:
`and`?

Reply:
I don't know if it's safe to omit the check when `sorted_index` is broadcasted by strides.
e.g. if `sorted_index` is considered an outer dimension with regarding to `inner_dim`, doesn't the broadcast scheme change that assumption?